docs/projects/fps/tutorial.md

```diff
@@ -41,13 +41,11 @@ To satisfy our design constraints, we need our health packs to contain some stat
 
 We write the definitions for these properties in [schemalang]({{urlRoot}}/reference/glossary#schema), a SpatialOS specific language. Let's do that now!
 
-**Step 1.** Using your file manager, navigate to `gdk-for-unity-fps-starter-project/`, then create a `schema` directory.
+**Step 1.** Inside the `gdk-for-unity-fps-starter-project/schema/` directory, create a `pickups` directory.
```

Comment:
this is something that the tech writers usually want. first say what tool we are using to get there. This is not necessary for the next steps as by then we already know we are using the file manager.
```suggestion
**Step 1.** Using your file manager, navigate to the `gdk-for-unity-fps-starter-project/schema/` directory, then create a `pickups` directory.
```